.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "python.formatting.provider": "black",
+  "python.formatting.blackArgs": ["--line-length", "120"],
+  "editor.formatOnSave": true,
+  "python.linting.flake8Enabled": true,
+  "python.linting.flake8Args": ["--max-line-length=120"]
+}
```

Comment:
Ya I think vscode just looks in the root of whatever directory you're in, so I always work in the root of the project - I'll move to `/backend` though since I've got that linting setup installed on my local anyways